src/algebra/algebra/tower.lean

```diff
@@ -212,8 +212,8 @@ variables (R) {S A B} [comm_semiring R] [comm_semiring S] [semiring A] [semiring
 variables [algebra R S] [algebra S A] [algebra R A] [algebra S B] [algebra R B]
 variables [is_scalar_tower R S A] [is_scalar_tower R S B]
 
-/-- Given a scalar tower `R`, `S`, `A` of algebras, reinterpret an `S`-subalgebra of `A` an as an
-`R`-subalgebra. -/
+/-- Given a tower `A / ↥U / S / R` of algebras, where `U` is an `S`-subalgebra of `A`, reinterpret
+`U` as an `R`--subalgebra of `A`. -/
```

Comment:
I notice `S`-subalgebra and `R`--subalgebra. It seems like they should be consistent (and single hyphen seems standard).

Reply:
Whoops, that was just a typo. Happy for me to set bors on this after fixing that?

Reply:
Yep -- I was even considering committing it quick myself and then merging.

bors d+